packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -28,7 +30,11 @@ import { getModifierKeyArray } from './utils';
 
 interface Props extends CommonInteractionProps {
   id?: string;
-  transform?: (rawSelection: Selection) => Selection;
+  transform?: (
+    rawSelection: Selection,
+    camera: Camera,
+    context: VisCanvasContextValue
```

Comment:
I'm now passing the camera and context to the `transform` callback so the consumer can use all the space-conversion functions without having to retrieve them manually. This is necessary for the story, since we're outside of `VisCanvas`.